src/jingle_sip/mod_jingle_sip_cets.erl

```diff
@@ -0,0 +1,232 @@
+%% @doc Backend module for mod_jingle_sip
+%% @author Michal Piotrowski <michal.piotrowski@erlang-solutions.com>
+%%
+%%==============================================================================
+%% Copyright 2018 Erlang Solutions Ltd.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%% http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%==============================================================================
+-module(mod_jingle_sip_cets).
+-behaviour(mod_jingle_sip_backend).
+
+-include("mongoose.hrl").
+
+-type call_id() :: binary().
+-type incoming_request() :: {node(), binary()}.
+-type outgoing_handle() :: binary().
+-type dialog_hangle() :: term().
+
+-export([init/2]).
+-export([set_incoming_request/5]).
+-export([set_incoming_handle/2]).
+-export([set_outgoing_request/4]).
+-export([set_outgoing_handle/4]).
+-export([set_outgoing_accepted/1]).
+-export([set_incoming_accepted/1]).
+-export([get_incoming_request/2]).
+-export([get_outgoing_handle/2]).
+-export([get_session_info/2]).
+-export([remove_session/1]).
+
+-ignore_xref([remove_session/1]).
+
+-record(jingle_sip_session, {sid, %% CallID
+                             dialog,
+                             state,
+                             direction,
+                             request,
+                             node,
+                             owner,
+                             from,
+                             to,
+                             now,
+                             meta}).
+
+-define(TABLE, cets_jingle_sip_session).
+
+-spec init(mongooseim:host_type(), gen_mod:module_opts()) -> ok.
+init(_Host, _Opts) ->
+    cets:start(?TABLE, #{keypos => 2}),
```

Comment:
```suggestion
    cets:start(?TABLE, #{keypos => #jingle_sip_session.sid}),
```

Reply:
Ok, I've moved record definition into a header file. 
CETS backend is written in a way, it does not know format of the record now.
So, we just use the keypos=2 (i.e. keypos for generic records).

Reply:
I just prefer to reference a particular field, so I don't have to check the record definition to see which field it is. If someone moves sid or adds a field before it (I don't know why, but if), then my version would remain valid. PLus, it would be self-descriptive without the need for a comment.